geth/rpc/call_raw.go

```diff
@@ -20,10 +25,36 @@ const (
 // thus, we will use zero ID as a workaround of this limitation
 var defaultMsgID = json.RawMessage(`0`)
 
+// All methods from the namespaces in this whitelist are
+// allowed in external RPCs
+var externalRPCNamespaceWhitelist = []string{
```

Comment:
This is kind of duplication of `APIModules` from `defaults.go`. We transform `APIModules` to the list.

Also, @mandrigin shouldn't we remove `shh` from `APIModules`?

Reply:
my idea was to have a separate PR for that

Reply:
I don't see how it's gonna work without `APIModules`. `APIModules` must be passed to the node config, otherwise, even if something will be whitelisted here, it will never reach the RPC server.